packages/styles/settings-tools/_s.buttons.scss

```diff
@@ -61,6 +48,20 @@ $button-border: get-border('m');
 
   /* see : https://github.com/philipwalton/flexbugs/issues/231 */
   height: 0;
+
+  .mc-button__icon {
+    @if map-get($sizeProps, 'icon-size') != $mu150 {
```

Comment:
why ?

Reply:
it would be `@if map-get($sizeProps, 'icon-size') != map-get($button-medium, 'icon-size') {`
but I think we will need those values if we use responsive modifiers right ?

Reply:
> why ?

The `$mu150` is the default size, so we don't need to override it with the same value.

Reply:
fixed